mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -72,6 +72,10 @@ public static string MapTimeZoneId (string id)
 					return "W. Europe Standard Time";
 				case "Canada/Eastern":
 					return "Eastern Standard Time";
+				case "Iran":
```

Comment:
This is an invalid time zone ID, it should be `Asia/Tehran`. Seems like it just works by accident.

Reply:
fixed